lang/ilias_nl.lang

```diff
@@ -1888,7 +1888,7 @@ chatroom_adm#:#client_settings#:#General Settings
 chatroom_adm#:#enable_osd#:#Enable On-Screen Notifications
 chatroom_adm#:#general_settings_title#:#General Chat Settings
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Refreshinterval (seconds)
+chatroom_adm#:#osd_interval#:#Refreshinterval (seconds)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.